src/api.rs

```diff
@@ -2503,6 +2505,8 @@ pub struct UpdateMonitorCheckIn {
     pub status: Option<MonitorCheckinStatus>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub duration: Option<u64>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub environment: Option<String>,
```

Comment:
Why is it `Option` if we have a default value?

Reply:
It's not absolutely required to send, this reflects that should the API be used anywhere else